### PR TITLE
test(root component) - added tests and minor fixes

### PR DESCRIPTION
**Pull request by valorkin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "es6-promise": "3.1.2",
     "es6-shim": "0.35.0",
     "es7-reflect-metadata": "1.6.0",
+    "intl": "1.1.0",
```
Comment (valorkin) [inline package.json:36]: @dydek please add `core-js` to package.json

**Comment by valorkin** [comment]
overall looks good, please wait for @rychkog 
so he can take a look too, later today

File: .config/test.bundle.js [hunk 2]
```diff
@@ -9,21 +9,35 @@
  * all here! Crazy huh. So we need to do some setup
  */
 Error.stackTraceLimit = Infinity;
-require('phantomjs-polyfill');
-require('es6-promise');
-require('es6-shim');
-require('es7-reflect-metadata/dist/browser');
+
+// Prefer CoreJS over the polyfills above
```
Comment (rychkog) [inline .config/test.bundle.js]: Comment seems to be irrelevant now

File: app/sections/root/sections/latest-posts/latest-posts.component.ts [hunk 3]
```diff
@@ -9,19 +9,20 @@ import {ContenfulContent} from '../../../../shared/services/contentful-content.s
...
   private limit: number = 3;
-
   private posts: Observable<ContentfulCommon<NodePageContent>[]>;
 
   constructor(private _contentfulContent: ContenfulContent) {
   }
 
   private toDate(dateAsString): Date {
-    return new Date(dateAsString);
+    if (!!dateAsString) {
```
Comment (rychkog) [inline app/sections/root/sections/latest-posts/latest-posts.component.ts:22]: if (dateAsString) { <--- Would be enough I guess

File: app/sections/root/sections/latest-posts/latest-posts.html [hunk 4]
```diff
@@ -7,7 +7,7 @@ <h2 class="post-head">Latest Blog Posts</h2>
           <img [contentful-src-id]="post.fields.thumbnail.sys.id"
                alt="{{post.fields.title}}">
         </div>
-        <div class="date-info">{{toDate(post.fields.createdAt) | date}}</div>
+        <div class="date-info">{{ toDate(post.fields.createdAt) | date }}</div>
```
Comment (rychkog) [inline app/sections/root/sections/latest-posts/latest-posts.html:10]: Formatter? :)

File: app/sections/root/sections/latest-videos/latest-videos.component.ts [hunk 5]
```diff
@@ -1,32 +1,31 @@
...
+export class LatestVideosComponent implements OnInit {
   @Input()
   private limit: number = 3;
-
   private videos: Observable<ContentfulCommon<NodePageContent>[]>;
 
   constructor(private _contentfulContent: ContenfulContent) {
   }
 
   private toDate(dateAsString): Date {
```
Comment (rychkog) [inline app/sections/root/sections/latest-videos/latest-videos.component.ts:23]: this one probably can be removed and substituted with the pipe you've created

**Comment by rychkog** [comment]
LGTM

File: app/sections/root/sections/latest-videos/latest-videos.component.ts (hunk 5 above)
Comment (dydek) [inline app/sections/root/sections/latest-videos/latest-videos.component.ts:23]: @rychkog yes, I'll replace it in the next PR

File: app/sections/root/sections/latest-posts/latest-posts.component.ts (hunk 3 above)
Comment (dydek) [inline app/sections/root/sections/latest-posts/latest-posts.component.ts:22]: yeah ... but I'll replace it with pipe anyway in the next PR

Comment (rychkog) [inline app/sections/root/sections/latest-posts/latest-posts.component.ts:22]: Cool then :)

File: app/sections/root/sections/latest-videos/latest-videos.component.ts (hunk 5 above)
Comment (rychkog) [inline app/sections/root/sections/latest-videos/latest-videos.component.ts:23]: :+1:
